easybuild/easyconfigs/t/TRIQS/TRIQS-2.1.1-foss-2018b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS'
+version = '2.1.1'
+
+homepage = 'https://triqs.github.io/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+"""
+
+docurls = ['https://triqs.github.io/triqs/2.1.x/reference.html']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/TRIQS/triqs/archive/']
+
+dependencies = [
+  ('Python', '2.7.15'),
+  ('Boost', '1.68.0', '-Python-2.7.15'),
+  ('Clang', '7.0.1'),
+  ('GMP', '6.1.2'),
+  ('Mako', '1.0.7', '-Python-2.7.15'),
+  ('h5py', '2.8.0', '-Python-2.7.15'),
+  ('CMake', '3.12.1')
+]
+
+sanity_check_paths = {
+    'files': ['lib/libtriqs.so'],
```

Comment:
Please avoid hardcoding `.so` (since on macOS it's actually `.dylib`), use the `SHLIB_EXT` constant instead:

```python
sanity_check_paths = {
    'files': ['lib/libtriqs.%s' % SHLIB_EXT],